### ISSUE TITLE
Krita doesn't actually work with some resource files missing in the package

### ISSUE THREAD

**Issue by wszqkzqk** [msys2/MINGW-packages#17612]
### Description / Steps to reproduce the issue

Run krita, and it will show:

```
You don't have any resource bundles enabled.
There are no brush presets available. Please enable a bundle that has presets before continuing.
```

To ignore these warnings to simply draw a picture will cause the crash of the program.

### Expected behavior

No missing resource files and works normally.

### Actual behavior

Continously shows errors.

### Verification

- [X] I have verified that my MSYS2 is up-to-date before submitting the report (see https://www.msys2.org/docs/updating/)

### Windows Version

MINGW64_NT-10.0-22621

### MINGW environments affected

- [ ] MINGW64
- [ ] MINGW32
- [X] UCRT64
- [ ] CLANG64
- [ ] CLANG32
- [ ] CLANGARM64

### Are you willing to submit a PR?

I'm not familiar with KDE and Qt toolchain

**Comment by wszqkzqk** [comment]
![image](https://github.com/msys2/MINGW-packages/assets/12961135/17a62f28-befa-4655-8282-7a6fd0976cb4)

**Comment by raedrizqie** [comment]
Arch adds this when configuring cmake:
`-DBUILD_KRITA_QT_DESIGNER_PLUGINS=ON`

**Comment by wszqkzqk** [comment]
> Arch adds this when configuring cmake: `-DBUILD_KRITA_QT_DESIGNER_PLUGINS=ON`

OK, I'll have a test.

**Comment by wszqkzqk** [comment]
> Arch adds this when configuring cmake: `-DBUILD_KRITA_QT_DESIGNER_PLUGINS=ON`

It doesn't work.

**Comment by jgbabyn** [comment]
I tried this on CLANGARM64 and ran into the same issue. But once I imported the krita 4 bundle from the source files it seems happy? Didn't test too extensively though. I think maybe the package as written doesn't import the bundles into the %APPDATA%\krita\ location it expects?

**Comment by Biswa96** [comment]
> But once I imported the krita 4 bundle from the source files it seems happy?

Thank you for looking into the issue. Could you provide the full path of those files?

**Comment by Biswa96** [comment]
Can anyone check if running krita with this command fixes the issue `krita --resource-location $MINGW_PREFIX/bin/data` ?

**Comment by jgbabyn** [comment]
Yes that works, I tested it after removing the %APPDATA%\krita folder which recreates the original error. Specifying the resource location as $MINGW_PREFIX/bin/data works.

**Comment by Biswa96** [comment]
It seems that changing the resource location with `--resource-location` option also writes some user generated files in `$MINGW_PREFIX/bin/data`. A different approach would be needed.
